src/main/java/com/cs56fitnessapp/models/Day.java

```diff
@@ -1,13 +1,55 @@
 package com.cs56fitnessapp.models;
 
 /**
- * @author Ksenia Koldaeva
- * Created: 9/17/17
- * Last Updated: 10/1/17
+ * @author Saori Shigehisa
+ * Created: 10/6/17
+ * Last Updated: 10/14/17
  */
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Date;
+
 
 public class Day implements ActivityFacts, DietFacts {
 
+    /** date */
+    private LocalDate date;
+
+    /** User */
+    private User user;
+
+    /** calorieGoal */
+    private double calorieGoal;
+
+    /** WorkoutList */
+    private ArrayList < Workout > workoutList;
+
+    /** MealList */
+    private ArrayList < Meal > mealList;
+
+    public Day (LocalDate date, User user, double caloriesGoal, ArrayList < Workout > workoutList, ArrayList < Meal > mealList){
+      this.date = date;
+      this.user = user;
+      this.calorieGoal = calorieGoal;
+      this.workoutList = workoutList;
+      this.mealList = mealList;
+    }
+
+
+
+    public boolean isToday() {
```

Comment:
Can be reduced to: 
`return now.getYear() == date.getYear() && now.getMonth() == date.getMonth() && now.getDayOfMonth() == date.getDayOfMonth();`

Reply:
Why should I delete this? 
Should I change back to "return false" which I made before?

Reply:
What I mentioned is not an error, you can leave this as it is. However code of this type is redundant - repeats itself: 
```java
if (a > b) {
  return true;
} else {
  return false;
}
```

This code is equivalent to the line: 
```java
return a > b;
```

`a > b` gets evaluated and the result gets returned.